winsup/doc/ntsec.xml

```diff
@@ -930,7 +930,16 @@ The two lines starting with the keywords <literal>passwd:</literal> and
 information from.  <literal>files</literal> means, fetch the information
 from the corresponding file in the /etc directory.  <literal>db</literal>
 means, fetch the information from the Windows account databases, the SAM
-for local accounts, Active Directory for domain account.  Examples:
+for local accounts, Active Directory for domain account.  For the current
+user, the default group is obtained from the current user token to avoid
+additional lookups to the group database. <literal>db-accuarte</literal> 
```

Comment:
db-accuarte --> db-accurate

Reply:
Since we do not build the documentation, this can easily be done in a (non-critical) follow-up PR. @placaze how about it?

Reply:
Thanks for catching that @placaze .  Created https://github.com/git-for-windows/msys2-runtime/pull/58 to fix it.